web/controllers/page_controller.ex

```diff
@@ -5,12 +5,23 @@ defmodule Healthlocker.PageController do
 
   def index(conn, _params) do
     featured_story = Post
-                     |> Post.find_single_story
-                     |> Repo.one
+                     |> Post.find_stories
+                     |> Repo.all
```

Comment:
do we need to _limit_ the number of stories we display?
(_we can always add this later when we have lots of content_...)